### DISCUSSION TITLE
roadmapping

### DISCUSSION THREAD

**Discussion by kmcdono2** [maps-as-data/MapReader#147] [Ideas]
- defining three groups: people using/exploring MapReader, people contributing to documentation, people contributing to code, people contributing to worked examples (see https://github.com/Living-with-machines/MapReader/issues/146)

- think about how these roles are separate, but also have some shared space (and we want to encourage the shared space, but also respect that there are different skills required)

- use roadmap doc created by @kallewesterling to review feedback vs our goals

- distinct problem of maps. this is something different, based on different principles from vector data capture.

- June: invite people to figure out what's missing

**Comment by kallewesterling** [comment]
[Roadmap document is available here](https://docs.google.com/document/d/1F3IqYuAupdwy3pDs1f7XCn_cGMwxkK2DtdO_G2OuWQY/edit)
